javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutNulls */
+/* list of values
+return all values except null
+figure out a way to remove null from array*/
+function filterOutNulls(values) {
+  var value = [];
+  for(var i = 0; i < values.length; i++) {
+    value.push(values[i]);
+    var noNull = value.filter(x => x !== null)
```

Comment:
If you can tell me exactly how this line works, you can keep it. Otherwise, please do this in a way you fully understand using the tools I have taught you.